src/en/administration/from_scratch.md

```diff
@@ -39,7 +39,9 @@ sudo -iu postgres psql -c "CREATE DATABASE lemmy WITH OWNER lemmy;"
 adduser lemmy --system --disabled-login --no-create-home --group
 ```
 
-Note that currently Lemmy only supports non-SSL connections to databases. More info [here](https://github.com/LemmyNet/lemmy/issues/3007).
+Note:  
+* Lemmy currently only supports non-SSL connections to databases. More info [here](https://github.com/LemmyNet/lemmy/issues/3007).
+* Your postgres config might need to be edited to allow password authentication instead of peer authentication. More info [here](https://github.com/LemmyNet/lemmy-docs/issues/220).
```

Comment:
Linking to an issue is not great. Better link to an external page which explains this, so that the issue can be closed.

Reply:
Agreed.... but the issue page gives an instant fix and solution to people who are not familiar with postgres. It took me some time too to understand from the various different search queries of "Failed Authentication postgres" what exactly was required to get things running.

Do you have a good reference with an example solution we can link directly to? All I'm seeing is various stackoverflow links

Reply:
I added the fix inline and removed the link to the issue. 
I've also just seen that there is a [Troubleshooting page](https://join-lemmy.org/docs/en/administration/troubleshooting.html), but it looks like it's more docker specific. Should I add the postgres note there instead?